### ISSUE TITLE
Extension Migration Plan

### ISSUE THREAD

**Issue by hyangah** [microsoft/vscode-go#3253]
We plan to change the extension's publisher ID from `ms-vscode` to `golang`.
The extension ID will become `golang.Go` after the transition.

_Date: TBD_

- [ ] Public announcement about the plan.
- [ ] Publish the final version of `ms-vscode.Go` (`0.14.N`). (#3247)@ramya-rao-a
- [ ] Notify extension pack authors who depend on `ms-vscode.Go`. @hyangah @stamblerre 
- [ ] (optional) Search GitHub for `.vscode/extensions.json` that mentions `ms-vscode.Go`, and notify the owners. @hyangah @stamblerre 
- [ ] Give @chrisdias publisher rights to golang publisher (needed for marketplace transfer). @hyangah 
- [ ] Contact the Microsoft/Marketplace team to transfer the extension. The actual transfer will take ~24hrs due to time zones. @chrisdias 
- [ ] Check all searches, etc. will work properly. Note a few exceptions:
      * The extension on disk will still have ms-vscode publisher ID until the new extension release.
      * VS Code extension recommendation will not work until the new extension release.
- [ ] Notify extension pack authors of the Marketplace transfer completion and encourage to publish updates with `golang.Go` id. @hyangah @stamblerre 
- [ ] Publish `v0.14.(N+1)` with `golang.Go`. It is identical with `v0.14.N` feature-wise.
- [ ] Update VS Code recommendations released to VS Code insiders/stable. @chrisdias 
- [ ] Remove @chrisdias from golang publisher. @hyangah 
- [ ] Archive github.com/microsoft/vscode-go. @ramya-rao-a

**Comment by DasJott** [comment]
Does this have something to do with it?
![image](https://user-images.githubusercontent.com/6298761/84249269-d46a9980-ab0a-11ea-9715-9d67a6ec45d1.png)
It appears, even though I have the extension installed... and ends up in this:
![image](https://user-images.githubusercontent.com/6298761/84249373-f401c200-ab0a-11ea-8d55-e27611dde97d.png)

**Comment by hyangah** [comment]
@DasJott Yes, that's a glitch due to this migration. The next VS Code version has a fix, and it will be released soon. Sorry for the inconvenience.

https://github.com/golang/vscode-go/issues/201#issuecomment-641549731

**Comment by DasJott** [comment]
No need to be sorry! That's not a huge thing ;)
Looking forward to the new version of VS Code then.

And as we are talking: Thank you very much for this great extension! I use it a lot every day and am happy that it exists!

**Comment by ramya-rao-a** [comment]
Thanks all! Archiving the repo...
